unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCSocialApiBridge.cs

```diff
@@ -80,7 +78,8 @@ private async UniTaskVoid InitializeSubscriptions(CancellationToken cancellation
         {
             cancellationToken.ThrowIfCancellationRequested();
 
-            // TODO: start listening to streams
+            // start listening to streams
+            UniTask.WhenAll(SubscribeToFriendshipEvents(cancellationToken));
```

Comment:
I think `WhenAll` is redundant here as you have one `UniTask` only, also as you don't wait for it change the returning type to `UniTaskVoid` and `Forget` it

Reply:
Done